composeApp/src/androidMain/kotlin/io/github/couchtracker/tmdb/TmdbMovie.kt

```diff
@@ -0,0 +1,35 @@
+package io.github.couchtracker.tmdb
+
+import app.cash.sqldelight.async.coroutines.awaitAsOneOrNull
+import app.moviebase.tmdb.model.AppendResponse
+import app.moviebase.tmdb.model.TmdbMovieDetail
+import app.moviebase.tmdb.model.TmdbReleaseDates
+import io.github.couchtracker.db.tmdb.TmdbCache
+
+/**
+ * Class that represents a TMDB movie.
+ * It holds no data, but provides the means to get all information,
+ * either by downloading them or loading them from cache.
+ */
+data class TmdbMovie(
+    val id: Int,
+    val language: TmdbLanguage,
+) {
```

Comment:
Question: did we decide not to go forward with having a `Movie` sealed interface or is it something that is coming later?

Reply:
We can do it later